README.md

```diff
@@ -28,8 +28,6 @@ Kotlin serialization consists of a compiler plugin, that generates visitor code
   * [Android](#android)
   * [Multiplatform (Common, JS, Native)](#multiplatform-common-js-native)
   * [Maven](#maven)
-* [Kotlin Serialization Guide](docs/serialization-guide.md)
-* [Full API reference](https://kotlin.github.io/kotlinx.serialization/)
```

Comment:
Maybe it's worth leaving it here even though it's a knit warning? I find it pretty useful

Reply:
Roman fixed this in master by moving these two bullets into 'Additional links' section outside of TOC